highfidelity/hifi-content
# Add app zone util

Set up: 
- Create a Zone entity
- Run this script on the zone entity
https://hifi-content.s3.amazonaws.com/robin/dev/utils/AddAppZone.js
- Ensure you have rez permissions in your domain (the fingerPaint app requires it)

Testing Plan:
1. Remove fingerPaint app from your tablet. This script uses it.
2. Enter zone entity, the fingerPaint app should now appear on your tablet
3. Feel free to draw with it
4. Leave zone entity, the fingerPaint app should be removed from the tablet and unload itself. 

Other tests:
- Teleport in and out of the zone entity
- After entering the zone, use Goto to change your domain. The fingerPaint app should be removed.
- Enter zone, delete fingerPaint app from your "running scripts...". Leave the zone. Enter the zone again, and the fingerPaint app should be loaded again.
